### DISCUSSION TITLE
Make the placeholder of the "Default Folder" field grayer

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#147] [General]
![](https://i.imgur.com/FuFoAqr.png)

As current, the color of the placeholder is strong enough that I thought `docs` is there, even though it's empty. Also, why not just put `docs` in there as the default as well? Then, if we add to the description that `docs` is the default and it's more convenient and clearer.

**Comment by Mara-Li** [comment]
Seems to be an issue with your theme. Normally, placeholder must use the `muted` color.

**Reply by ooker777** [reply]
what about the idea of just putting `docs` as the default value?

**Reply by Mara-Li** [reply]
Don't like it, prefer forcing the user to know the settings and don't messing up with their repo

**Reply by ooker777** [reply]
Why doesn't this still help them to understand the setting? Also, what do you mean by "don't messing up with their repo"? Do you mean you don't want you to mess up with their repo, or you don't want them to accidentally mess up with their repo?
